### PR TITLE
Support to import int data in test framework

### PR DESCRIPTION
**Pull request by yixinglu**
Use config.yaml to share same csv data files

### CODE REVIEW

File: tests/common/csv_import.py [hunk 1]
```diff
@@ -22,20 +23,64 @@ class CSVImporter:
...
+                header[self._key(prop)] = self._prop(prop, name)
+        else:
+            raise ValueError("Invalid config file")
+        self._header = [header[str(i)] for i in range(len(header))]
+
+    @staticmethod
+    def _vid(vid):
+        vtype = vid['type']
+        if 'function' not in vid:
+            return f":VID({vtype})"
```
Comment (yixinglu) [inline tests/common/csv_import.py:73]: srcdst vid

File: tests/common/csv_import.py [hunk 2]
```diff
@@ -22,20 +23,64 @@ class CSVImporter:
...
+                tagname = tag['name']
+                for prop in tag['props']:
+                    header[self._key(prop)] = self._prop(prop, tagname)
+        elif file_desc["type"] == "edge":
+            edge = file_desc["edge"]
+            name = edge['name']
+            src = edge['srcVID']
+            header[self._key(src)] = self._vid(src)
+            dst = edge['dstVID']
+            header[self._key(dst)] = self._vid(dst)
```
Comment (yixinglu) [inline tests/common/csv_import.py:62]: rank

File: tests/common/utils.py [hunk 3]
```diff
@@ -334,24 +336,23 @@ def exec(stmt):
...
-                continue
-            stmts.append(ln)
-        rs = sess.execute(' '.join(stmts))
+    config_path = os.path.join(data_dir, 'config.yaml')
+
+    with open(config_path, 'r') as f:
+        config = yaml.full_load(f)
+        schemas = config['schema']
+        stmts = ' '.join(
+            filter(lambda x: not x.startswith("--"),
```
Comment (yixinglu) [inline tests/common/utils.py:348]: need not filter

File: tests/common/csv_import.py [hunk 4]
```diff
@@ -22,20 +23,64 @@ class CSVImporter:
...
         self._insert_stmt = ""
         self._create_stmt = ""
+        self._file_desc = file_desc
         self._type = None
+        self._gen_header(file_desc)
 
     def __iter__(self):
         with open(self._filepath, 'r') as f:
             for i, row in enumerate(csv.reader(f)):
                 if i == 0:
```
Comment (yixinglu) [inline tests/common/csv_import.py:37]: withheader

File: tests/common/csv_import.py [hunk 5]
```diff
@@ -22,20 +23,65 @@ class CSVImporter:
...
+                    yield self.parse_header(header)
                 else:
                     yield self.process(row)
 
+    def _gen_header(self, file_desc):
+        header = {}
+        if file_desc["type"] == "vertex":
+            vertex = file_desc["vertex"]
+            vid = vertex["vid"]
+            header[self._key(vid)] = self._vid(vid, "VID")
```
Comment (laura-ding) [inline tests/common/csv_import.py:49]: use the var to save the "VID"

File: tests/tck/conftest.py [hunk 6]
```diff
@@ -153,13 +156,16 @@ def execution_should_be_succ(graph_spaces):
 def raised_type_error(err_type, time, msg, graph_spaces):
     res = graph_spaces["result_set"]
     assert not res.is_succeeded(), "Response should be failed"
+    err_type = err_type.strip()
+    msg = msg.strip().replace('$', '\$')
+    res_msg = res.error_msg()
     if res.error_code() == ErrorCode.E_EXECUTION_ERROR:
-        expect_msg = f"{msg.strip()}"
-        assert err_type.strip() == "ExecutionError"
+        assert err_type == "ExecutionError"
```
Comment (laura-ding) [inline tests/tck/conftest.py:163]: the `E_EXECUTION_ERROR`, doesn't have the error message "ExecutionError"

**Review by nevermore3 — APPROVED** [review]

File: tests/common/csv_import.py (hunk 5 above)
Comment (yixinglu) [reply tests/common/csv_import.py]: fixed

**Review by laura-ding — APPROVED** [review]

**Review by nevermore3 — APPROVED** [review]

**Pull request merged**